api/graphite.go

```diff
@@ -507,7 +513,7 @@ func findTreejson(query string, nodes []idx.Node) models.SeriesTree {
 }
 
 func (s *Server) metricsDelete(ctx *middleware.Context, req models.MetricsDelete) {
-	peers := cluster.Manager.MemberList()
+	peers := cluster.Manager.MemberList(false, true)
```

Comment:
if this calls a non-ready node, won't that node just respond with a bad error message and then this whole request results in a failure? or is this what we want because then the delete wasn't complete?

Reply:
Yeah we want to send to all nodes to make sure the request can be as complete as it can be.
it resulting in errors is a separate issue -> #1249